proparse/src/main/java/org/prorefactor/treeparser01/TP01Support.java

```diff
@@ -1197,21 +1197,22 @@ private ITable astTableLink(AST tableAST) {
     return buffer.getTable();
   }
 
-  private SymbolScopeSuper classStateSuper(JPNode classNode, String className) throws RefactorException {
-    LOG.trace("Entering classStateSuper {} {}", classNode, className);
+  private TreeParserSuperSymbolScope classStateSuper(JPNode classNode, String className) throws RefactorException {
+    return null;
+    /* LOG.trace("Entering classStateSuper {} {}", classNode, className);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)